eva/utils/timer.py

```diff
@@ -17,38 +17,29 @@
 
 from eva.utils.logging_manager import logger
 
-# GLOBAL CONSTANTS
-_START_TIME = None
-_END_TIME = None
 
+class Timer:
+    """Class used for logging time metrics.
 
-def start_timer():
-    global _START_TIME
-    global _END_TIME
+    This is not thread safe"""
```

Comment:
What are the implications of this -- like when not to use this? We could maybe add that as a comment.

Reply:
There isn't an example in the system right now. I added a comment so that in the future when we bring ray, we are careful.